### PR TITLE
Brisk.app

### PR DESCRIPTION
**Pull request by keith**
This is our long lived develop branch with a pretty decent MVP of Brisk.

### CODE REVIEW

File: Brisk/StoryboardRouter.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        }
+
+        self.windowController?.window?.delegate = self
+        NSApp.activate(ignoringOtherApps: true)
+    }
+}
+
+extension StoryboardRouter: NSWindowDelegate {
+    func windowWillClose(_ notification: Notification) {
+        StoryboardRouter.sharedRouter.windowController = nil
```
Comment (sberrevoets) [inline Brisk/StoryboardRouter.swift]: `self.`?

File: Brisk/StoryboardRouter.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import AppKit
+
+final class StoryboardRouter: NSObject {
+    fileprivate static var sharedRouter = StoryboardRouter()
```
Comment (sberrevoets) [inline Brisk/StoryboardRouter.swift]: `let`?

File: Brisk/StoryboardRouter.swift (hunk 1 above)
Comment (keith) [inline Brisk/StoryboardRouter.swift]: done

File: Brisk/StoryboardRouter.swift (hunk 2 above)
Comment (keith) [inline Brisk/StoryboardRouter.swift]: done

**Pull request merged**